### ISSUE TITLE
Reduce minimum interval limit to 1 minute

### ISSUE THREAD

**Issue by waqasy** [huishun98/Cron#1]
Can we reduce interval limit to 1 minute instead of 5 minutes?

P.S. great app. love the simplicity.

**Comment by MaazArfi** [comment]
Yes you can but taking care of your requirements and accordingly.

**Comment by ProtocolNebula** [comment]
Hi @huishun98 , please check my PR that can solve this Issue.

I'm using for personal use and it works, anyways, a refactor to a better precission api should be nice in a future versions.
